src/pages/open-source.js

```diff
@@ -21,23 +20,40 @@ const OpenSource = ({ data }) => {
       seoMetaDescription,
       featuredCaseStudy,
       statement,
+      talksSectionImage,
+      talksSectionTitle,
+      talksSectionTalks,
+      talkSectionCtaText,
+      talksSectionCtaLink,
       technologyPartnersSectionTitle,
-      technologyPartners
+      technologyPartners: partners
     }
   } = data
 
+  const talks = talksSectionTalks.filter(({ type }) => type === 'Talk')
+
   return (
     <Layout>
       <Head page={{ title, seoTitle, seoMetaDescription }} />
       <CaseStudyPreview isTop caseStudy={featuredCaseStudy} />
       <Statement>{statement}</Statement>
-      <OpenDeliverables {...data} />
```

Comment:
We need this in here

Reply:
Oh. Makes sense. I imported it and it was telling me `OpenDeliverables` is not used anywhere so I deleted 😆  I will put back

Reply:
:D